app/terms/page.js

```
@@ -0,0 +1,528 @@
+        if (visibleEntries.length > 0) {
+          const closest = visibleEntries.reduce((prev, curr) => {
+            return Math.abs(curr.boundingClientRect.top) < Math.abs(prev.boundingClientRect.top) ? curr : prev;
+          });
+          setActiveSection(closest.target.id);
+        }
+      },
+      { rootMargin: "0px 0px -60% 0px", threshold: 0 }
+    );
+
+    const sectionElements = document.querySelectorAll("section[id]");
+    sectionElements.forEach((el) => observer.observe(el));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)